src/metatrain/experimental/soap_bpnn/model.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, atomic_types: List[int], n_layer: int) -> None:
 
 class SoapBpnn(torch.nn.Module):
 
-    __supported_devices__ = ["multi-gpu", "cuda", "cpu"]
```

Comment:
I thought it does?

Reply:
Yes, but in a different way. To access multi-GPU training with SOAP-BPNN you have to launch multiple processes, each with its own `mtt train options.yaml` (for example with srun), and each using `cuda`. Perhaps we can improve the naming

Reply:
What? Why is this not mentioned anywhere in the docs. Also why do I make the effort and rewrite these messages if none of the models is supporting multi GPUs? 

On top I think that the renaming of user inputs i.e. `multi-gpu` to `multi-cuda` in the code as you wrote it is a very bad idea.

Reply:
How to do distributed training for SOAP-BPNN is explained in the docs. And yes, renaming things has its downsides. You can open an issue for it